### ISSUE TITLE
Virtualbox shared folder have hard-to-satisfy requirements

### ISSUE THREAD

**Issue by aspiwack** [NixOS/nixops#375]
I'm using Nixops on a non-NixOs host. When I try to enable shared folders for Virtualbox, using for instance the example from the documentation:
```
deployment.virtualbox.sharedFolders.home = {  hostPath = "/home";  readOnly = false; };
```
I get the following error during configuration building:
```
cp: cannot stat '/nix/store/1b0biv3pwppayn0n567mf8vrna7mjwh6-VirtualBox-GuestAdditions-5.0.6-4.1.13/sbin/mount.vboxsf': No such file or directory
builder for ‘/nix/store/qgnk0l71z53qfg9xa1y18wghq1193c63-extra-utils.drv’ failed with exit code 1
```

**Comment by rbvermaa** [comment]
This was fixed in 523242771715a250dff5e34a67327f9e64881b7d.

**Comment by rbvermaa** [comment]
I will create a new release (1.3.1) for nixops next week. Until then, please build nixops from source.
